### ISSUE TITLE
[BUG]? Python bids-validator returns true on non-bids folder names 

### ISSUE THREAD

**Issue by bendhouseart** [bids-standard/legacy-validator#1821]
Was playing around a bit last week and noticed that the python version of the validator seems to misclassify non-bids folder names as compliant.

See:

```python
>>> from bids_validator import BIDSValidator
>>> validator = BIDSValidator()
>>> validator.is_bids('/sub_07/eeg/sub-07_task-matchingpennies_events.tsv')
True
>>>
```

Python and validator version(s):
```python
Python 3.10.9 (main, Mar  1 2023, 12:20:14) [Clang 14.0.6 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import bids_validator
>>> bids_validator.__version__
'1.12.0'
```

**Comment by effigies** [comment]
I'm a little inclined to mark this wontfix. If you would like to do filename validation in Python, the bidsschematools validator is going to be better supported.

**Comment by effigies** [comment]
Actually, I think the better fix is to depend on `bidsschematools` and write shims for the current API. We should strip the Python validator out of the JS validator repository and free up JS developers to reorganize without potentially breaking the Python packaging.

**Comment by sappelhoff** [comment]
+1 to remove the python validator from the JS repo and potentially even deprecating it in favor of the new schema based python validator.

**Comment by effigies** [comment]
This will be fixed with #1967 
```python
In [1]: from bids_validator import BIDSValidator

In [2]: BIDSValidator().is_bids('/sub_07/eeg/sub-07_task-matchingpennies_events.tsv')
Out[2]: False
```
